cmd/wio/toolchain/npm/resolve/types.go

```diff
@@ -22,11 +23,14 @@ type Info struct {
 }
 
 type Node struct {
-    name string
-    ver  string
-    deps []*Node
-
-    resolve *semver.Version
+    Name            string
+    ConfigVersion   string
+    ResolvedVersion *semver.Version
+    Dependencies    []*Node
+
+    Vendor bool
+    Path   string
+    Config *types.PkgConfig
```

Comment:
Just make sure that you check if this is `nil`, which means the dependency was not retrieved